### ISSUE TITLE
How to pass flag unstable_native_nodejs_ts_config?

### ISSUE THREAD

**Issue by krutoo** [microsoft/vscode-eslint#2074]
Hi, in docs of ESLint there is flag `unstable_native_nodejs_ts_config` to make it possible to use TS config

How can I pass this flag to VSCode ESLint server?

Docs:
https://eslint.org/docs/latest/use/configure/configuration-files#native-typescript-support

**Comment by krutoo** [comment]
Looks like it is possible via `eslint.options`:

```json
{
  "eslint.options": {
    "flags": ["unstable_native_nodejs_ts_config"]
  }
}
```

But now I have error: 

```
The unstable_native_nodejs_ts_config flag is enabled, but native TypeScript support is not enabled in the current Node.js process. You need to either enable native TypeScript support by passing --experimental-strip-types or remove the unstable_native_nodejs_ts_config flag.
```

**Comment by krutoo** [comment]
#### The final configruation to use ESLint 9 in VScode with Node.js 24 and native TS support

Content of `./vscode/settings.json` in project folder

```json
{
  "eslint.useFlatConfig": true,
  "eslint.runtime": "<path of your node binary, can be defined by `which node`>",
  "eslint.execArgv": ["--experimental-strip-types"],
  "eslint.options": {
    "flags": ["unstable_native_nodejs_ts_config"]
  }
}
```
